MultimodalQnA/ui/gradio/multimodalqna_ui_gradio.py

```diff
@@ -69,9 +69,9 @@ def add_text(state, textbox, audio, request: gr.Request):
         state.append_message(state.roles[1], None)
         state.skip_next = False
         return (state, state.to_gradio_chatbot(), None, None) + (disable_btn,) * 1
+    # If it is a image query
```

Comment:
This shouldn't be an `elif` but a regular `if` because audio and image should be allowed in the same query.